eq-author-api/schema/tests/multipleChoiceAnswer.test.js

```diff
@@ -145,7 +145,10 @@ describe("multiple choice answer", () => {
       });
 
       it("should resolve options", () => {
-        expect(last(queriedAnswer.options).id).toEqual(last(answer.options).id);
+        expect(queriedAnswer.options).toHaveLength(2);
+        expect(
+          last(queriedAnswer.options.filter(o => !o.mutuallyExclusive)).id
+        ).toEqual(last(answer.options.filter(o => !o.mutuallyExclusive)).id);
```

Comment:
Change looks good, but can the test be re-worked to not perform the filtering in the test itself? I think it would be clearer.